Saule/Queries/Pagination/PaginationQuery.cs

```diff
@@ -26,6 +31,12 @@ public PaginationQuery(PaginationContext context)
             PreviousPage = isNumber && page > 0
                 ? CreateQueryString(context.ClientFilters, page - 1)
                 : null;
+            if (value is IPagedResult && context.PerPage.GetValueOrDefault(0) != 0)
+            {
+                var paged = (IPagedResult)value;
+                var totalPages = (int)Math.Ceiling((double)paged.TotalResultsCount / context.PerPage.Value);
+                LastPage = CreateQueryString(context.ClientFilters, totalPages);
```

Comment:
`LastPage` is `CreateQueryString`? Either the method name is wrong or the property has the wrong value, I think?

Reply:
Mabe it should be `GenerateLastPageParameters`

Reply:
Yeah I was just commenting on the method name. `CreateQueryString` seems like a bigger thing than just the value for the last page. I'm also curious why `LastPage` etc are strings, but that's another issue and I think not something you introduced here.

Reply:
yeah. i didn't touch that part as it was part of existing logic